applications/GeoMechanicsApplication/custom_utilities/transport_equation_utilities.hpp

```diff
@@ -0,0 +1,65 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#pragma once
+
+// Project includes
+
+// Application includes
+#include "geo_mechanics_application_variables.h"
+
+namespace Kratos
+{
+
+class GeoTransportEquationUtilities
+{
+public:
+    using VectorType = Vector;
+    using MatrixType = Matrix;
```

Comment:
I haven't found any usages of these aliases, so I would suggest to remove them.

Reply:
Thank you. Removed.